src/MSBuildLocator/MSBuildLocator.cs

```diff
@@ -43,14 +44,6 @@ public static class MSBuildLocator
         /// </summary>
         public static bool IsRegistered => s_registeredHandler != null;
 
-        /// <summary>
-        ///     Allow discovery of .NET SDK versions that are unlikely to be successfully loaded in the current process.
-        /// </summary>
-        /// <remarks>
-        ///     Defaults to <see langword="false"/>. Set this to <see langword="true"/> only if your application has special logic to handle loading an incompatible SDK, such as launching a new process with the target SDK's runtime.
-        /// </remarks.
-        public static bool AllowQueryAllRuntimeVersions { get; set; } = false;
```

Comment:
No concerns about this being a breaking change? I don't know what your API compat model is; I could imagine in your case it's just an app picking a version and that's fine.

Reply:
We've avoided breaking changes (at least against published-to-nuget.org versions) so far; we should probably keep doing so. I guess allow both?

(We should also be accumulating a list of breaking changes, I have no compunction against releasing a 2.0 with a cleaner but breaking API.)

Reply:
Sounds good! Feel free to contribute https://github.com/microsoft/MSBuildLocator/issues/282